src/acore-wp-plugin/src/Components/UnstuckMenu/UnstuckView.php

```
@@ -31,36 +37,34 @@ public function getUnstuckmenuRender($chars)
+                                    <div class="acore-char-row" style="<?= esc_attr($clsStyle) ?>">
+                                        <span class="acore-char-name"><?= esc_html($char["name"]) ?></span>
+                                        <span class="acore-char-meta">
+                                            <span class="acore-level" data-exp="<?= AcoreCharColors::expansionSlug(intval($char["level"])) ?>" title="<?= esc_attr(AcoreCharColors::expansionLabel(intval($char["level"]))) ?>">Level <?= intval($char["level"]) ?></span>
+                                            <img class="race-icon" height="32" width="32" title="<?= esc_attr(AcoreCharColors::getRaceName(intval($char["race"]))) ?>" src="<?= esc_url(ACORE_URL_PLG . "web/assets/race/" . intval($char["race"]) . (intval($char["gender"]) == 0 ? "m" : "f") . ".webp") ?>">
+                                            <img class="class-icon" height="32" width="32" title="<?= esc_attr(AcoreCharColors::getClassName(intval($char["class"]))) ?>" src="<?= esc_url(ACORE_URL_PLG . "web/assets/class/" . intval($char["class"]) . ".webp") ?>">
+                                            <button
+                                                class="unstuck-button"
+                                                data-char-name="<?= esc_attr($char["name"]) ?>"
+                                                <?= $isDisabled ? 'disabled' : '' ?>
+                                                title="<?= esc_attr($tooltipText) ?>">
+                                                <img src="<?php echo ACORE_URL_PLG . 'web/assets/unstuck/hearthstone.jpg'; ?>" alt="Unstuck">
```

PHPStan (2.2.2)

[error] Constant ACORE_URL_PLG not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)